modules/custom/moj_resources/src/Utilities.php

```diff
@@ -18,25 +18,35 @@ class Utilities {
     *
     * @param int $prisonId
     * @param int[] $prisonCategories
-    * @param QueryInterface
+    * @param QueryInterface $query
+    * @param bool $isTerm
```

Comment:
If you try and query on a non-existent field the query blows up, added this switch.